src/applications/legalpad/engineextension/PhabricatorLegalpadSignaturesSearchEngineAttachment.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+final class PhabricatorLegalpadSignaturesSearchEngineAttachment
+  extends PhabricatorSearchEngineAttachment {
+
+  public function getAttachmentName() {
+    return pht('Document signers');
+  }
+
+  public function getAttachmentDescription() {
+    return pht('Get the signer list for the project.');
+  }
+
+  public function willLoadAttachmentData($query, $spec) {
+    $query->needSignatures(true);
+  }
+
+  public function getAttachmentForObject($object, $data, $spec) {
+    $signers = array();
+    foreach ($object->getSignatures() as $signature) {
+      $signatures[] = array(
+        'phid' => $signature->getPHID(),
+        'signerPHID' => $signature->getSignerPHID(),
+        'exemptionPHID' => $signature->getExemptionPHID(),
+        'isExemption' => $signature->getIsExemption(),
+        'signerName' => $signature->getSignerName(),
+        'signerEmail' => $signature->getSignerEmail(),
```

Comment:
Can you get the date signed in here as well? I

Reply:
that's a good idea!

Reply:
added